docs/source/our_stack.rst

```diff
@@ -10,31 +10,104 @@ The most up-to-date reference on our codebase is our code itself.
     To see when any documentation page was last updated, click "Edit on GitHub" in
     the top right, then see the timestamp on the last commit.
 
-...
-TODO(Kevin): stub out the rest
-
-Overview
+Gameplay
 --------
+Gameplay is where our strategy is produced. It takes in the current state and
+assigns all robots a task based on some coordination. Our
+system utilizes a play selector that chooses a play based on world state. After
+that play assigns roles, each role selects a tactic based on their progress, and
+tactic select an atomic skill to perform. The gameplay loop is set to tick at
+60Hz. The code block below describes this process. Each time the timer
+ends, the callback method is entered:
 
-ROS2
-----
+.. code-block:: python
 
-Vision
-------
+        timer_period = 1 / 60  # seconds
+        self.timer = self.create_timer(timer_period, self.gameplay_tick)
 
-Gameplay
---------
+Motion Control
+--------------
+A lot of the motion control is done on robot by the firmware, but part of the
```

Comment:
This is a little vague imo. I would have put that both are PID controllers to correct for error, but the PID controller in software corrects for position error and the PID in FW corrects velocity.